samples/beta/get-model-evaluation.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+function main(
+  projectId = 'YOUR_PROJECT_ID',
+  location = 'us-central1',
+  modelId = 'YOUR_MODEL_ID',
+  modelEvaluationId = 'YOUR_MODEL_EVALUATION_ID'
+) {
+  // [START automl_video_classification_get_model_evaluation_beta]
```

Comment:
I'm a little confused by what's happening with the region tags.  I see this one 5 times below.

Reply:
Yep, we are using the `START` `END` region tag magic to only include parts of the code on the docs site for the one line difference so that we don't have to maintain so many identical samples.